bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI/appsettings.AT21.json

```diff
@@ -0,0 +1,18 @@
+{
+  "PlatformSettings": {
+    "ApiAccessManagementEndpoint": "https://platform.at21.altinn.cloud/accessmanagement/api/v1/",
```

Comment:
The endpoints that is not needed can be removed

Reply:
ok